dell-c6320/workflows/dell-secure-erase-drive-graph.json

```diff
@@ -0,0 +1,82 @@
+{
+  "friendlyName": "Secure Erase Drive",
+  "injectableName": "Graph.Drive.SecureErase",
+  "options": {
+    "finish-bootstrap-trigger": {
+      "triggerGroup": "secureErase"
+    },
+    "drive-secure-erase": {
+      "eraseSettings": null
```

Comment:
Since no schema is defined for secure task, so this is still safe at present, But this will potentially violate the task schema if we define a schema for it in future.

I know that developer has kept a habit to set a `null` value inside graph options as something like placeholder to tell user that this option need be pay more attention. I will think about whether to keep this habit.

No action for you for this comment. Just some notice for you.